src/sentry/integrations/github/integration.py

```diff
@@ -833,38 +890,43 @@ def get_app_url(self) -> str:
 
     def dispatch(self, request: HttpRequest, pipeline: Pipeline) -> HttpResponseBase:
         with record_event(IntegrationPipelineViewType.GITHUB_INSTALLATION).capture() as lifecycle:
+            self.active_user_organization = determine_active_organization(request)
+
+            if self.active_user_organization is not None and features.has(
+                "organizations:github-multi-org",
+                organization=self.active_user_organization.organization,
+                actor=request.user,
+            ):
+                error_page = self.check_pending_integration_deletion(request=request)
```

Comment:
Do the same "if any installations are in pending deletion land block the installation attempt" check up here, since we're 'exiting' early if we have a `chosen_installation_id`